doc feedback for client generation (#1811)
Azure/typespec-azure: website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx

}

@doc("Read my resource")

> op Get is ResourceRead<InputModel>;

Reviewer: ```suggestion
op get is ResourceRead<InputModel>;
```

while there should change to follow naming convention

package com.petstorenamespace;
public final class PetStoreNamespaceClient {
    public Response<BinaryData> getWithResponse(String name, RequestOptions requestOptions)
    public InputOptions get(String name)
}
```

</TabItem>
</Tabs>

### Renaming operations and parameters

> Simillarly, you can rename operations like in the example below:

Reviewer 2: Nit:
```suggestion
Similarly, you can rename operations like in the example below:
```

</TabItem>
</Tabs>

> You cannot at this moment rename paramters in the client.tsp file.

Reviewer 2: I'd add more to this and show an example of having the `@clientName` decorator over a parameter in an example operation. And say:
```suggestion
You cannot at this moment rename parameters in the client.tsp file. You will need to add the `@clientName` decorator over the parameter directly, example:

```
